docs/INSTALLATION.md

```diff
@@ -63,19 +63,13 @@ Below are instructions for install using the most common method - using Homebrew
      $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
      ```
 
-2. Tap the [Homebrew Cask](https://caskroom.github.io/) — this allows us to install pre-built binaries like the JDK.
-
-     ```
-     $ brew tap adoptopenjdk/openjdk
-     ```
-
-3.  Install the JDK:
+2.  Install the JDK:
 
      ```
-     $ brew install --cask adoptopenjdk11
+     $ brew install --cask temurin
```

Comment:
Same question here, can we make sure students install Java 21 specifically, instead of the latest version? According to [the adoptium installation instructions](https://adoptium.net/installation/) this should be the way to do so:

```sh
$ brew tap homebrew/cask-version
$ brew install --cask temurin21
```

Reply:
Thanks! Sorry for missing that.